caffe2/python/tutorials/MNIST.ipynb

```diff
@@ -59,23 +61,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Looks like you ran this before, so we need to cleanup those old files...\n",
-      "DB:  /home/zhecao/caffe2_notebooks/tutorial_data/mnist/mnist-train-nchw-leveldb\n",
-      "Database exists already. Delete the folder if you have issues/corrupted DB, then rerun this.\n",
-      "DB:  /home/zhecao/caffe2_notebooks/tutorial_data/mnist/mnist-test-nchw-leveldb\n",
-      "Database exists already. Delete the folder if you have issues/corrupted DB, then rerun this.\n",
-      "training data folder:/home/zhecao/caffe2_notebooks/tutorial_data/mnist\n",
-      "workspace root folder:/home/zhecao/caffe2_notebooks/tutorial_files/tutorial_mnist\n"
-     ]
-    }
-   ],
```

Comment:
This looks like more than a link update. I just wanted to make sure you mean to remove this much of the notebook in this particular pull request.